model/riscv_platform.sail

```diff
@@ -311,6 +311,24 @@ function clint_store(addr, width, data) = {
     mtimecmp = vector_update_subrange(mtimecmp, 63, 32, sail_zero_extend(data, 32)); /* FIXME: Redundant zero_extend currently required by Lem backend */
     clint_dispatch();
     MemValue(true)
+  } else if addr == MTIME_BASE & 'n == 8 then {
+    if   get_config_print_platform()
+    then print_platform("clint<8>[" ^ BitStr(addr) ^ "] <- " ^ BitStr(data) ^ " (mtime)");
+    mtime = zero_extend(data);
```

Comment:
Do you actually need the `zero_extend()`s? In theory Sail could figure out that you don't... but maybe it isn't smart enough?

Reply:
No, Sail is indeed able to figure this out. I removed them